src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
+interface TodoListProps {
+  todos: Todo[];
+  selectedTodo: Todo | undefined;
+  onSelect: (todo: Todo) => void;
+}
+
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  selectedTodo: { id: selectedId } = { selectedId: 0 },
```

Comment:
The destructuring of `selectedTodo` with a default value `{ selectedId: 0 }` is incorrect. It should be `selectedTodo` instead of `selectedId` to match the expected structure of the `Todo` type. Consider using `selectedTodo?.id` for optional chaining.